packages/core/src/constants.ts

```diff
@@ -8,11 +8,12 @@ export const VERSION = packageVersion[0] + '.' + packageVersion[1] + '.0' // Ver
 
 const PROTOCOL_NAME = 'hopr'
 
-export const PROTOCOL_STRING = `/${PROTOCOL_NAME}/msg/${VERSION}`
-export const PROTOCOL_ACKNOWLEDGEMENT = `/${PROTOCOL_NAME}/ack/${VERSION}`
-export const PROTOCOL_PAYMENT_CHANNEL = `/${PROTOCOL_NAME}/payment/open/${VERSION}`
-export const PROTOCOL_ONCHAIN_KEY = `/${PROTOCOL_NAME}/onChainKey/${VERSION}`
-export const PROTOCOL_HEARTBEAT = `/${PROTOCOL_NAME}/heartbeat/${VERSION}`
+export const PROTOCOL_STRING = (network, hoprChannelContractAddress) =>
```

Comment:
Think this should use the network identifier rather than  the address.

Also UPPERCASE is for constants, not functions.